recipe/meta.yaml

```diff
@@ -8,21 +8,23 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 9dfb18b568729d0219f758cddca1a91bab59f62ca41ee0e8acce5e657ec97b6c
+  sha256: 5e07b23d70ceed64f3bb36ae4b9abc52354c16c98d45ab037bee2b5fbffe586c
 
 build:
+  skip: True  # [py<35]
```

Comment:
There is no evidence that the package is not compatible with earlier python versions. There are classifiers that specify python 3.6, but that's not a hard requirement: https://github.com/python-trio/trustme/blob/v0.9.0/setup.py#L37-L41

However, `cryptography` has a `py<37`, so that should probably be the upper bound: https://github.com/AnacondaRecipes/cryptography-feedstock/blob/master/recipe/meta.yaml#L13

Reply:
This has been updated